src/openforms/submissions/exports.py

```diff
@@ -61,6 +62,13 @@ def export_submissions(
     return response
 
 
+def xml_serialize_value(value) -> str:
+    if isinstance(value, (list, dict)):
+        return json.dumps(value)
```

Comment:
I don't like that suddenly we have JSON inside of XML and require XML parsers to also parse JSON.

For an export with singular values, we now get this:

```xml
<submissions>
  <submission>
    <field name="Formuliernaam">Camunda demo sprint 11</field>
    <field name="Inzendingdatum">2022-01-17T15:18:02.140744</field>
    <field name="bedrag">500</field>
    <field name="invoiceCategory">Misc</field>
  </submission>
</submissions>
```

which is okay. But the multiple variant would result in:

```xml
<submissions>
  <submission>
    <field name="Formuliernaam">Camunda demo sprint 11</field>
    <field name="Inzendingdatum">2022-01-17T15:18:02.140744</field>
    <field name="bedrag">[500, 600]</field>
    <field name="invoiceCategory">["Misc", "Staff"]</field>
  </submission>
</submissions>
```

how does a parser determine what to parse as JSON and what to parse as string?

I'd much prefer the following output which sticks to XML semantics:

```xml
<submissions>
  <submission>
    <field name="Formuliernaam"><value>Camunda demo sprint 11</value></field>
    <field name="Inzendingdatum"><value>2022-01-17T15:18:02.140744</value></field>
    <field name="bedrag">
        <value>500</value>
        <value>600</value>
    </field>
    <field name="invoiceCategory">
        <value>Misc</value>
        <value>Staff</value>
    </field>
  </submission>
</submissions>
```

* this normalizes that `field` has at least one `value` element
* this opens up the path to add type information in the future / multi information: e.g. `<field name="bedrag" type="int" multiple="true">`